modules/azure/src/main/java/org/testcontainers/azure/AzuriteContainer.java

```diff
@@ -0,0 +1,175 @@
+     */
+    public AzuriteContainer(final DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+        withExposedPorts(DEFAULT_BLOB_PORT, DEFAULT_QUEUE_PORT, DEFAULT_TABLE_PORT);
+    }
+
+    /**
+     * Configure SSL with a custom certificate and password.
+     *
+     * @param pfxCert  The PFX certificate file
+     * @param password The password securing the certificate
+     * @return this
+     */
+    public AzuriteContainer withSsl(final MountableFile pfxCert, final String password) {
+        cert = pfxCert;
+        pwd = password;
+        certExtension = ".pfx";
+        return this;
+    }
+
+    /**
+     * Configure SSL with a custom certificate and private key.
+     *
+     * @param pemCert The PEM certificate file
+     * @param pemKey  The PEM key file
+     * @return this
+     */
+    public AzuriteContainer withSsl(final MountableFile pemCert, final MountableFile pemKey) {
+        cert = pemCert;
+        key = pemKey;
+        certExtension = ".pem";
+        return this;
+    }
+
+    /**
+     * Sets the hostname we want to use to connect to our emulator. (default: {@link #DEFAULT_HOST})
+     *
+     * @param host The host name
+     * @return this
+     */
+    public AzuriteContainer withHost(final String host) {
+        this.host = host;
+        return this;
+    }
+
+    @Override
+    protected void configure() {
+        withCommand(getCommandLine());
+        if (cert != null) {
+            logger().info("Using path for cert file: '{}'", cert);
+            withCopyFileToContainer(cert, "/cert" + certExtension);
+            if (key != null) {
+                logger().info("Using path for key file: '{}'", key);
+                withCopyFileToContainer(key, "/key.pem");
+            }
+        }
+    }
+
+    /**
+     * Returns the connection string for the default credentials.
+     *
+     * @return connection string
+     */
+    public String getDefaultConnectionString() {
+        return getConnectionString(WELL_KNOWN_ACCOUNT_NAME, WELL_KNOWN_ACCOUNT_KEY);
+    }
+
+    /**
+     * Returns the connection string for the account name and key specified.
+     *
+     * @param accountName The name of the account
+     * @param accountKey  The account key
+     * @return connection string
+     */
+    public String getConnectionString(final String accountName, final String accountKey) {
+        final String protocol = cert != null ? "https" : "http";
+        return String.format(
+            CONNECTION_STRING_FORMAT,
+            protocol,
+            accountName,
+            accountKey,
+            protocol,
+            host,
+            getMappedPort(DEFAULT_BLOB_PORT),
+            accountName,
+            protocol,
+            host,
+            getMappedPort(DEFAULT_QUEUE_PORT),
+            accountName,
+            protocol,
+            host,
```

Comment:
I think it should be getHost() instead

```suggestion
            getHost(),
            getMappedPort(DEFAULT_BLOB_PORT),
            accountName,
            protocol,
            getHost(),
            getMappedPort(DEFAULT_QUEUE_PORT),
            accountName,
            protocol,
            getHost(),
```